examples/intermediate/06_history_backup.py

```
@@ -0,0 +1,538 @@
+요구사항: OPENAI_API_KEY 환경 변수
+"""
+
+import os
+import json
+import sqlite3
+from datetime import datetime
+from pathlib import Path
+from typing import List, Optional
+from pyhub.llm import LLM
+from pyhub.llm.types import Message
+from pyhub.llm.history import InInMemoryHistoryBackup
```

Ruff (0.11.9)

`pyhub.llm.history.InInMemoryHistoryBackup` imported but unused

Remove unused import: `pyhub.llm.history.InInMemoryHistoryBackup`

(F401)